docs/astro.config.ts

```diff
@@ -19,95 +32,97 @@ export default defineConfig({
+            //             { label: '1. Step',  slug: 'introduction'},
+            //             { label: '2. Step',  slug: 'introduction'},
+            //         ],
+            // },
             {
                 label: 'Components',
                 items: [
-                    {
-                        label: 'Account Management',
-                        items: [
-                            { label: 'Overview',  slug: 'introduction'},
-                            { label: 'Tagging',  slug: 'introduction'},
-                            { label: 'Budgets',  slug: 'introduction'},
-                            { label: 'SOP - Create Account',  slug: 'introduction'},
-                            { label: 'SOP - Add Account & Bootstrap',  slug: 'introduction'},
-                        ]
-                    },
-                    {
-                        label: 'Networking',
-                        items: [
-                            { label: 'Overview',  slug: 'introduction'},
-                            { label: 'VPCs',  slug: 'introduction'},
-                            { label: 'NATs',  slug: 'introduction'},
-                            { label: 'Bastion Hosts',  slug: 'introduction'},
-                            { label: 'VPC Peering',  slug: 'introduction'},
-                        ]
-                    },
-                    {
-                        label: 'Security',
-                        items: [
-                            { label: 'Overview',  slug: 'introduction'},
-                            { label: 'Service Control Policies',  slug: 'introduction'},
-                            { label: 'Permission Boundary',  slug: 'introduction'},
-                            { label: 'SecurityHub',  slug: 'introduction'},
-                            { label: 'Control Tower Controls',  slug: 'introduction'},
-                            { label: 'AWS Config',  slug: 'introduction'},
-                            { label: 'AWS Guard Duty',  slug: 'introduction'},
-                            { label: 'AWS Macie',  slug: 'introduction'},
-                        ]
-                    },
-                    {
-                        label: 'Identity',
-                        items: [
-                            { label: 'Overview',  slug: 'introduction'},
-                            { label: 'Iam Identity Center',  slug: 'introduction'},
-                        ]
-                    },
-                    {
-                        label: 'Data Services',
-                        items: [
-                            { label: 'Overview',  slug: 'introduction'},
-                            { label: 'LakeFormation',  slug: 'introduction'},
-                        ]
-                    },
+                    // {
+                    //     label: 'Account Management',
+                    //     items: [
+                    //         { label: 'Overview',  slug: 'introduction'},
```

Comment:
What do you think about still having these as TODOs? So people can have an overview on different parts and know that it's going to be updated eventually.